oinksignup.js

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from "react";
+// refresh token
+import { Redirect } from "react-router-dom";
+import { connect } from "react-redux";
+import { signIn } from "../../redux/actions/auth";
+import Avatar from '@material-ui/core/Avatar';
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+
+function Copyright() {
+  return (
+    <Typography variant="body2" color="textSecondary" align="center">
+      {'Copyright © Oink '}

```

Comment:
just for future reference, bracket notation is not needed for strings within an element! :) but good use of it in the next line!

Reply:
thank you for pointing that out! let me resend a new pull request....